lte/gateway/c/core/oai/tasks/ngap/include/ngap_client_servicer.h

```diff
@@ -44,6 +49,10 @@ class NGAPClientServicer : public NGAPClientServicerBase {
   NGAPClientServicer(NGAPClientServicer const&) = delete;
   void operator=(NGAPClientServicer const&) = delete;
 
+  map_string_string_t map_ngapState_tableKey_protoStr;
```

Comment:
avoid camelCase naming to be consistent

Reply:
additional question on this, what is the plan for this mapping? are we saving the serialized proto message on this? if so, what is the purpose? from looking at the code it seems that this is only used for unit testing scenarios...

Reply:
- Addressed review comment
- In the unit test, after the serialization process we do not write to the Redis DB, instead we write to an in-memory data store map defined in the map.h and NgapClientServicer.